numerapi/numerapi.py

```diff
@@ -1242,7 +1242,7 @@ def stake_get(self, username: str) -> float:
         return stake
 
     def stake_change(self, nmr, action: str = "decrease",
-                     model_id: str = None) -> Dict:
+                     model_id: str = None, tournament_number=8) -> Dict:
```

Comment:
please add a type hint, like `tournament_number: int = 8`

Reply:
And rename `tournament_number` to just `tournament`. I agree that `tournament_number` is the better naming, however, the signature should follow what we do for the other methods and there it's called `tournament`

Reply:
Thanks for the suggestions. I've made the changes, please review